### PR TITLE
PYIC-4572: Update RevokeVcsHandler and RestoreVcsHandler READMEs

### PR DESCRIPTION
**Pull request by MikeCollingwood**
### What changed

- Updated RevokeVcsHandler and RestoreVcsHandler READMEs

### Why did it change

- Adds clarity to the two new lambdas

### Issue tracking

- [PYIC-4555](https://govukverify.atlassian.net/browse/PYIC-4555)

[PYIC-4555]: https://govukverify.atlassian.net/browse/PYIC-4555?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: lambdas/restore-vcs/README.md [hunk 1]
```diff
@@ -1,14 +1,40 @@
...
+
+### Payload
+
+- The payload is the same as for the revoke lambda.
+
+### Command
 
 ```bash
-aws-vault exec <profile> -- aws lambda invoke --function-name restore-vcs-<env> --invocation-type RequestResponse --payload fileb://restoreVcsLambdaPayload.json response.json
+aws-vault exec PROFILE -- \
```
Comment (lwebb-iig) [inline lambdas/restore-vcs/README.md:24]: Maybe worth providing a way of doing this with [SSO](https://govukverify.atlassian.net/wiki/spaces/PLAT/pages/3554280423/Getting+Ready+for+AWS+SSO) as I think this is the way we're expected to interact with AWS profiles and sessions moving forward?

File: lambdas/revoke-vcs/README.md [hunk 2]
```diff
@@ -1,19 +1,96 @@
...
+- This lambda works on user credentials in the `user-issued-credentials-v2-ENVIRONMENT` table ([e.g.](https://eu-west-2.console.aws.amazon.com/dynamodbv2/home?region=eu-west-2#item-explorer?table=user-issued-credentials-v2-build)).
+- The lambda archives during revocation in the `revoked-user-credentials-ENVIRONMENT` table ([e.g.](https://eu-west-2.console.aws.amazon.com/dynamodbv2/home?region=eu-west-2#item-explorer?table=revoked-user-credentials-build)).
+- In the case of failures which can have audit events, we send the audit event `IPV_VC_REVOKED_FAILURE`.
+- In the case where we cannot send audit events, we break the loop through the list.
+
+---
+
+## Running the lambda
+
+> This lambda should not be run by anyone without the explicit sign-off of stakeholders.
```
Comment (lwebb-iig) [inline lambdas/revoke-vcs/README.md]:
```suggestion
> This lambda should not be invoked in production without explicit sign-off from stakeholders
```

File: lambdas/restore-vcs/README.md [hunk 3]
```diff
@@ -1,14 +1,40 @@
...
+
+This lambda mirrors the functionality of [RevokeVcsHandler](lambdas/revoke-vcs/README.md), except for some differences:
+1. The direction of change of VC ownership is opposite
+2. The lambda does not send a failing audit event
+
+---
+
+## Running the lambda
+
+> This lambda should only be called with explicit permission from stakeholders
```
Comment (lwebb-iig) [inline lambdas/restore-vcs/README.md]:
```suggestion
> This lambda should not be invoked in production without explicit sign-off from stakeholders
```

**Review by Joe-Edwards-GDS — APPROVED** [review]

File: lambdas/restore-vcs/README.md (hunk 1 above)
Comment (MikeCollingwood) [reply lambdas/restore-vcs/README.md:24]: I think you're right though I think I will merge this now, but it's helpful to know this and I may go back and update it

**Pull request merged**